simtbx/diffBragg/src/diffBragg_gpu_kernel.cu

```diff
@@ -372,19 +372,19 @@ void gpu_sum_over_steps(
                 MAT3 Amat = UBO;
                 MAT3 Ainv = UBO.inverse();
                 if (s_gamma_miller_units){
-                  anisoG = anisoG * Amat;
+                  anisoG = anisoG * Bmat_realspace;
```

Comment:
modifiying global matrix each call?

Reply:
Moved this outside the threaded region and using a local copy. OK?